canonical/snapcraft.io
# fix: Fix issue with saving settings when navigating from listing

## Done
Fixed bug where settings changes wouldn't save if navigating from the listings page

## How to QA
- Go to https://snapcraft-io-4913.demos.haus/<SNAP_NAME>/listing
- Click to the "Settings" tab
- Check that changes can be saved successfully

## Testing
- [] This PR has tests
- [x] No testing required (explain why): Existing tests in place

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-17094
